compiler/noirc_evaluator/src/ssa/plonky2_gen/mod.rs

```diff
@@ -217,6 +217,7 @@ impl P2Type {
                         P2Type::Integer(bit_size, false)
                     }
                 }
+                NumericType::Signed { bit_size } => P2Type::Integer(bit_size, true),
                 _ => {
```

Comment:
This is now unreachable. Please, remove.

Reply:
Done: https://github.com/blocksense-network/noir/pull/81/commits/167e7b8e44198dd2c7dea9f18c5583b268b34398